credit/datasets/load_dataset_and_dataloader.py

```diff
@@ -5,13 +5,11 @@
     MultiprocessingBatcher,
     MultiprocessingBatcherPrefetch
 )
-from credit.data import ERA5_and_Forcing_Dataset
 from torch.utils.data import DataLoader
 from torch.utils.data.distributed import DistributedSampler
 from credit.transforms import load_transforms
-from credit.parser import credit_main_parser, training_data_check
-from credit.datasets import setup_data_loading, set_globals
```

Comment:
import setup_data_loading should be restored since it is used in the load_dataset function.

Reply:
Good catch! Thanks.